src/logging.c

```diff
@@ -44,6 +46,13 @@ const char * g_rcutils_log_severity_names[] = {
   [RCUTILS_LOG_SEVERITY_FATAL] = "FATAL",
 };
 
+enum rcutils_colorized_output
+{
+  RCUTILS_COLORIZED_OUTPUT_FORCE_ENABLE,
+  RCUTILS_COLORIZED_OUTPUT_FORCE_DISABLE,
+  RCUTILS_COLORIZED_OUTPUT_DEFAULT,
```

Comment:
May I suggest to use:

* `RCUTILS_COLORIZED_OUTPUT_FORCE_DISABLE` implicit value: `0` (matching env var value)
* `RCUTILS_COLORIZED_OUTPUT_FORCE_ENABLE` implicit value: `1` (matching env var value)
* `RCUTILS_COLORIZED_OUTPUT_AUTO` implicit value: `2`